docs/book/intro.md

```diff
@@ -82,7 +82,7 @@ $twitter = new Laminas\Twitter\Twitter($config);
 Make sure you substitute the values you discovered earlier in the
 configuration before attempting to connect.
 
-> #### Note
+> ### Note on SSL certificate verification
```

Comment:
We do notes at a level 4 heading. You can use markdownlint-ignore-next-line annotations before the line to ignore this linting error.

Reply:
@weierophinney 
> We do notes at a level 4 heading.

We have level 3 everywhere!

Reply:
Ouch, I STAND CORRECTED!

So the only thing to revert is the removal of the `$` bash prefixes.